controllers.py

```diff
@@ -125,6 +134,7 @@
     "casskop-operator": ["cassandracluster", "cassandrarestore", "cassandrabackup"],
     "xtradb-operator": ["perconaxtradbcluster", "perconaxtradbclusterbackup", "perconaxtradbclusterrestore", "perconaxtradbbackup"],
     "yugabyte-operator": ["ybcluster"],
+    "nifikop-operator": ["nificluster"],
```

Comment:
I noticed that there are multiple different CRDs for the nifikop, but I guess we only care about the nificluster for now, right?

Reply:
Yes